src/Browscap/Command/RewriteDivisionsCommand.php

```diff
@@ -0,0 +1,934 @@
+                            $currentPlatform      = ['name' => $platformProperties['Platform'], 'major-version' => $split[0], 'minor-version' => $split[1], 'key' => $platformkey];
+                        } elseif (false !== mb_strpos($currentPlatform['key'], 'WinXPb') && false !== mb_strpos($platformkey, 'WinXPa')) {
+                            ++$currentChunk;
+                            $chunk[$currentChunk] = [json_encode($platformkey)];
+                            $currentPlatform      = ['name' => $platformProperties['Platform'], 'major-version' => $split[0], 'minor-version' => $split[1], 'key' => $platformkey];
+                        } elseif (false !== mb_strpos($currentPlatform['key'], 'WinXPa') && false !== mb_strpos($platformkey, 'WinXPb')) {
+                            ++$currentChunk;
+                            $chunk[$currentChunk] = [json_encode($platformkey)];
+                            $currentPlatform      = ['name' => $platformProperties['Platform'], 'major-version' => $split[0], 'minor-version' => $split[1], 'key' => $platformkey];
+                        } elseif ($platformProperties['Platform'] !== $currentPlatform['name']
+                            || $split[0] !== $currentPlatform['major-version']
+                        ) {
+                            ++$currentChunk;
+                            $chunk[$currentChunk] = [json_encode($platformkey)];
+                            $currentPlatform      = ['name' => $platformProperties['Platform'], 'major-version' => $split[0], 'minor-version' => $split[1], 'key' => $platformkey];
+                        } elseif (is_numeric($platformProperties['Platform_Version']) && $split[1] > $currentPlatform['minor-version']) {
+                            ++$currentChunk;
+                            $chunk[$currentChunk] = [json_encode($platformkey)];
+                            $currentPlatform      = ['name' => $platformProperties['Platform'], 'major-version' => $split[0], 'minor-version' => $split[1], 'key' => $platformkey];
+                        } else {
+                            $chunk[$currentChunk][] = json_encode($platformkey);
+                        }
+                    }
+
+                    $childData['platforms'] = $chunk;
+                }
+            }
+
+            try {
+                $normalized = $twig->render('division.json.twig', ['divisionData' => $divisionData]);
+            } catch (\Twig_Error_Loader | \Twig_Error_Runtime | \Twig_Error_Syntax $e) {
+                $logger->critical($e);
+
+                continue;
+            }
+
+            file_put_contents($file->getPathname(), $normalized);
+        }
+
+        $output->writeln('Done');
+
+        return 0;
+    }
+
+    /**
+     * @param array $platforms
+     *
+     * @return array
+     */
+    private function sortPlatforms(array $platforms)
+    {
+        $platformVersions = [];
+
+        $allplatforms = [
```

Comment:
Is it possible to somehow use the `platforms.json` file for this listing, and maintain the sorting of the platforms in that file?

Seems like a bit of a pain if we have to update this list whenever a new platform is added/removed/changed.

Reply:
Will try to change this.